syscalls.go

```diff
@@ -469,3 +471,40 @@ var haveBatchAPI = internal.FeatureTest("map batch api", "5.6", func() error {
 	}
 	return nil
 })
+
+var haveProbeReadKernel = internal.FeatureTest("bpf_probe_read_kernel", "5.5", func() error {
+	spec := ProgramSpec{
```

Comment:
It doesn't look like this `ProgramSpec` is actually passed to anything, could it be removed?

Reply:
Sure!